Formula/opencsg.rb

```diff
@@ -15,19 +14,17 @@ class Opencsg < Formula
   depends_on "qt5" => :build
   depends_on "glew"
 
-  # This patch adds support for specifying INSTALLDIR
-  # It has been submitted upstream and accepted 20160709, through private email
-  # (as that's how submissions are done)
-  # Should be in the next release (> 1.4.0)
+  # This patch disabling building examples
```

Comment:
I've reported the ability to disable examples as a feature request upstream.
How would you go about building without patches? The build system simply doesn't properly build examples. We could of course patch how the examples are being built, but there isn't, IMO, any need to build examples for homebrew distribution.